src/deck.py

```diff
@@ -1,7 +1,27 @@
 from card import Card
+import random
+SUITS = ["C", "D", "H", "S"]
+RANKS = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"]
 
-
+#TODO: implement ranking
 class Deck:
     def __init__(self):
-        print("Deck constructor")
-        self._card = Card()
+        self.cards = []
+        for suit in (SUITS):
+            for rank in (RANKS):
+                card = Card(rank, suit)
+                self.cards.append(card.to_json)
+                
+        random.shuffle(self.cards)
+    
+    def pop(self):
+        return self.cards.pop()
+    
+    def draw(self, num):
```

Comment:
This probably isn't ideal. I'd recommend keeping the deck list fixed once it's shuffled, and keep a pointer or a counter to track the "top" of the deck. That way you'll have access to the values of the previously dealt cards, in case you need them for some reason.